x-pack/solutions/search/plugins/search_playground/public/utils/select_indices.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type React from 'react';
+import type { EuiSelectableOption } from '@elastic/eui/src/components/selectable/selectable_option';
+
+export const handleSelectOptions =
```

Comment:
Curious, why do we need to this function ? 
When an index is selected, wouldn't updated options list have indices that are checked & unchecked? is it needed to do these manually ?